### DISCUSSION TITLE
Can this be used without a framework?  With just Javascript and jQuery?

### DISCUSSION THREAD

**Discussion by redit0** [khmyznikov/pwa-install#77] [Q&A] [answered]
If so, I would love it if someone could give me some pointers on how to do so.  Thanks in advance!!

**Comment by khmyznikov** [comment, marked as answer]
Sure! Take a look at this html: https://github.com/khmyznikov/pwa-install/blob/main/docs/index.html Not all code there is meant to be used in a real app, but I think you'll figure out.
